src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRInfoViewModel.swift

```diff
@@ -8,7 +8,7 @@ import UIKit
 struct ExposureSubmissionQRInfoViewModel {
 
 	init(supportedCountries: [Country]) {
-		countries = supportedCountries
+		countries = supportedCountries.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
```

Comment:
Can we put this in an Extension for [Country] ? 

`extension Array where Element : Country `

Something along those lines.

Reply:
Cool idea 👍 
Done with 8058e6e884154897883606e6c2f95fb1b772a5bb